template_sets.go

```diff
@@ -68,6 +83,8 @@ func NewSet(name string, loaders ...TemplateLoader) *TemplateSet {
 		name:          name,
 		loaders:       loaders,
 		Globals:       make(Context),
+		tags:          make(map[string]*tag),
+		filters:       make(map[string]FilterFunction),
```

Comment:
Gave this a spin - having empty filters leads to panic when redering variables with escaping, since `escape` filter is not present. Think `NewDefaultSet` will have to include at least that one!

Reply:
@draganm It might be a good idea to use newSet instead of NewDefaultSet. Users can only use NewSet which includes all default filters and tags to create a special template set. What do you think of this idea?

Reply:
Yeah, that works, but as I said, `NewDefaultSet` has this huge dependency list of things that need to be provided, that it makes is quite niche. Upon reflection, probably just a word of warning in the GoDoc would be probably enough to point people to `NewSet`